ArcGISRuntimeSDKQt_CppSamples/Analysis/StatisticalQueryGroupSort/StatisticalQueryGroupSort.cpp

```diff
@@ -240,12 +243,18 @@ void StatisticalQueryGroupSort::removeOrderBy(const QString& field)
 void StatisticalQueryGroupSort::removeOrderBy(int index)
 {
   // remove the order by at a given index
-  m_orderBys.removeAt(index);
-  emit orderBysChanged();
+  if(!m_orderBys.empty())
```

Comment:
There are a few of these. I won't comment on them all but please use consistent time.